kernel/src/net/socket/netlink/common/unbound.rs

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use core::marker::PhantomData;
+
+use crate::{
+    events::IoEvents,
+    net::socket::{
+        netlink::{
+            common::bound::BoundNetlink, table::SupportedNetlinkProtocol, GroupIdSet, MessageQueue,
+            MessageReceiver, NetlinkSocketAddr,
+        },
+        util::datagram_common,
+    },
+    prelude::*,
+    process::signal::Pollee,
+};
+
+pub struct UnboundNetlink<P: SupportedNetlinkProtocol> {
```

Comment:
`UnboundNetlink` does not seem to be used outside `common/`. Can it be `pub(super)`?